commands/scale_cluster.go

```diff
@@ -243,10 +236,23 @@ func scaleClusterRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
 	var desiredCapacity int64
 
```

Comment:
I cannot comment on the line I spotted above. 

```
		if n == 0 || n == 1 {
			autoScalingEnabled = true
		} else {
			autoScalingEnabled = false
		} 
```

There is no reason for the `else` clause since the zero value is already `false`.

Reply:
Fixed.